cmd/trayscale/systray.go

```diff
@@ -1,6 +1,43 @@
 package main
 
-import "fyne.io/systray"
+import (
+	"fyne.io/systray"
+)
+
+type tray struct {
+	showItem *systray.MenuItem
+	quitItem *systray.MenuItem
+}
+
+func initTray(online bool) *tray {
+	systray.SetIcon(statusIcon(online))
+	systray.SetTitle("Trayscale")
+
+	showWindow := systray.AddMenuItem("Show", "")
+	systray.AddSeparator()
+	quit := systray.AddMenuItem("Quit", "")
+
+	return &tray{
+		showItem: showWindow,
+		quitItem: quit,
+	}
+}
+
+func (t *tray) NotfiyQuit() <-chan struct{} {
+	return t.quitItem.ClickedCh
+}
+
+func (t *tray) NotfiyShow() <-chan struct{} {
+	return t.showItem.ClickedCh
+}
+
+func (t *tray) SetIcon(online bool) {
```

Comment:
This does set the icon, but if a menu item is added to allow toggling the status directly from the menu, I suspect that this same method will also update that, so I'd change the name to something along the lines of `SetOnlineStatus()` or something more general.